setup.py

```diff
@@ -40,7 +30,24 @@
     package_data={"": ["template_metafile.yml"]},
     include_package_data=True,
     python_requires=">=3.7",
-    install_requires=install_requires,
+    install_requires=[
+        "pynwb",
+        "tqdm",
```

Comment:
IMO this is perhaps a bit _too_ loose of relaxation since there are, for many of these, absolute minimal versions required to do even the most basic things in `nwb-conversion-tools`. An example is schema formation from `hdmf` docvals, which requires a function that wasn't added until a particular version of `hdmf`.

I also prefer to have these extracted from a `requirements-*.txt` file so that we always know internally that requirements files are the ones that need to be updated any time a version bump or new dependency is needed; historically it's easy to forget to also update the setup every time you need to add something.

Reply:
I'd be fine with having these as `requirements-min.txt`. And yes, we should specify minimum requirements for ones where that is needed.